src/components/Article.jsx

```diff
@@ -0,0 +1,33 @@
+import { useParams } from "react-router-dom";
+import { getArticleById } from "../utils/api";
+import { useState, useEffect } from "react";
+
+const Article = () => {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState({});
+  const date = new Date(article.created_at).toLocaleDateString();
+
```

Comment:
Would be nice to see a loading state.  When I picked this PR up. Your API wasn't responding so your frontend showed nothing at all. A little loading message would be nice to see 😄